Algorithms/Divide-and-Conquer/closest_pair_of_points.cpp

```diff
@@ -0,0 +1,171 @@
+// C++ Program to Find Closest Pair of Points
+// among the given set of points using
+// Divide and Conquer Approach.
+
+// Author: Pritam Negi.
+
+#include <iostream>
+#include <cfloat>
+#include <cstdlib>
+#include <cmath>
+using namespace std;
+ 
+// Declaration of structure
+// for x any y coordinate of a point
+struct Point
+{
+    int x, y;
+};
+ 
+// Function to sort array of points according to x coordinate
+int compareX(const void* a, const void* b)
```

Comment:
@pritamnegi Completely copied from [GeeksforGeeks](https://www.geeksforgeeks.org/closest-pair-of-points/)

Reply:
@pritamnegi please change it